api/python/cellxgene_census/src/cellxgene_census/_open.py

```diff
@@ -76,6 +86,8 @@ def open_soma(
     Args:
         census_version:
             The version of the Census, e.g. "latest" or "stable". Defaults to "stable".
+        mirror:
+            The mirror used to retrieve the Census. Defaults to the `us-west-2` mirror.
```

Comment:
Defaults to _AWS S3 us-west-2_  (i.e., need to document the provider, not just the region)

Reply:
@pablo-gar do we want to introduce a naming convention for the mirrors?

Reply:
A great question.

That said, I don't think you need a naming scheme to fix this docstring issue.

Reply:
For now this suffices: "[provider] [region]" e.g. "AWS S3 us-west-2"